xapian-letor/feature/feature.cc

```diff
@@ -94,4 +95,44 @@ Feature::set_collection_termfreq(const std::map<std::string,
     collection_termfreq = collection_tf;
 }
 
+std::map<std::string, Xapian::termcount>
+Feature::get_termfreq() const
+{
+    LOGCALL(API, (std::map<std::string, Xapian::termcount>),
+	    "Feature::get_termfreq", NO_ARGS);
```

Comment:
There's no need to wrap long `LOGCALL` lines - they're not really intended for humans to read, and `xapian-check-patch` has an exception in its line length check for them.

(Ideally we'd have a script to automatically add them so they were complete and automatically ensured to be up-to-date - we quite often find someone failed to update the `LOGCALL` when a method signature changes - often that means a build error in a logging build, but it's possible for it to introduce subtle bugs when logging is enabled.)